Philippines - House: Add term 17

octopusinvitro opened the pull request:
This PR adds data for the new term (term 17) in the Philippines

Checkbox list with links here:
https://gist.github.com/octopusinvitro/f748e068d2249978389bab1cb0edff18

Preserve old data *if necessary* (ep-data)
----------------------------------------

* [x] data no longer available upstream is in manual file <https://github.com/everypolitician/everypolitician-data/blob/master/data/Philippines/House/sources/manual/term-16.csv>
* [x] idmaps and reconciliation files preserved too <https://github.com/everypolitician/everypolitician-data/blob/master/data/Philippines/House/sources/idmap/term-16.csv>
* [x] redundant scraper no longer used in `instructions.json` <https://github.com/everypolitician/everypolitician-data/blob/master/data/Philippines/House/sources/instructions.json#L3-L7>
* [x] clean build works with no changes <https://github.com/everypolitician/everypolitician-data/pull/19107#issuecomment-255760085>


New term (ep-data)
------------------

These are in the PR to add the new data

* [x] a single `terms.csv` exists and has previous terms *and* this new term in 
* [x] previous term end date is correct, and source credited in commit message
* [x] new term start date is correct, and source credited in commit message


New data (ep-data)
------------------

* [x] new memberships are merged or reconciled
* [x] party groupings are correct (screenshot) --> I left a screenshot but I am not sure about 1-CARE and SAGIP


Reconciliations against other sources
------------------------------------

* [x] wikidata is being reconciled (or else an issue exists for it)

To run the site locally with these changes: 
```
DATASOURCE='https://cdn.rawgit.com/everypolitician/everypolitician-data/d11d067833c1c285c6f3832791f0314e9c634e1f/countries.json' foreman start
```

## New term
![terms](https://cloud.githubusercontent.com/assets/2157089/19695414/167bfd44-9adb-11e6-9d9c-89ca483ae72f.png)

## Party groupings
![parties](https://cloud.githubusercontent.com/assets/2157089/19725046/54e95d42-9b7b-11e6-85a7-6c54c14f8053.png)

octopusinvitro commented:
Summary of changes in `data/Philippines/House/ep-popolo-v1.0.json`:

## People

### Added



- `62717031-beb0-44b8-b894-37777c617ed4` - Abayon, Harlin Neil III J.

- `acc2853f-7e76-49fb-8184-5e7282bcc7dc` - Acosta, Gil "Kabarangay" P.

- `226421ae-f958-4289-8a8b-22e3a5f57cc1` - Aggabao, Ma. Lourdes R.

- `0082ce21-03e7-4f0b-b837-0229590f6fb2` - Akbar, Jum J.

- `a88eb0c4-bc8f-432e-9645-e04c868b3b66` - Almario, Joel Mayo Z.

- `373a9abb-aed4-461b-a42f-ea2b34103c58` - Alonte-Naguiat, Len B.

- `7a820239-e3a9-4b21-8fb1-e8cbba7c2b7e` - Alvarez, Pantaleon D.

- `e012ef3c-0acd-4d88-b274-c93dda9e6a76` - Antonio, Michelle M.

- `abd7a941-a106-494c-8244-0fdb9cab77f2` - Arbison, Munir M.

- `9fe58df4-f6f2-4aea-8b5a-da89fba4dfff` - Arcillas, Arlene B.

- `285a8439-3477-44b4-896b-945c36781b61` - Austria Barzaga, Jennifer

- `7b3162b2-0d2e-46db-ba97-22af8f43fc0d` - Bagatsing, Cristal L.

- `7ee62ce0-3450-40e0-aabe-1211516c824b` - Barbers, Robert Ace S.

- `d77a2ee9-859c-41ba-853c-725836ef876f` - Bautista-Bandigan, Lorna P.

- `60dce242-08c5-4b6a-ae52-662d8d9c9cb7` - Belaro, Salvador Jr. B.

- `8f97c112-56f0-4690-be47-206c611cf39b` - Belmonte, Ricardo Jr. T.

- `9f2c1150-7553-44b9-afcd-a9065384cc79` - Bernos, Joseph Sto. Niño B.

- `4356189b-3575-4c5f-bb11-2d662886a6ed` - Bertiz, Aniceto "John" III D.

- `eefc4b51-3986-4817-a02f-b3fe0054a940` - Biazon, Rozzano Rufino B.

- `ab1cdda3-b74d-4246-a726-4623776eb7e7` - Billones, Emmanuel A.

- `7cd865f0-8b4f-464c-bcc2-a21240bf4144` - Biron, Ferjenel G., M.D.

- `7ab80a30-8dc5-44b6-b4df-24cdab6cc883` - Bolilia, Lianda B.

- `c5560722-9c7c-45d4-bc0a-aec7039efb7a` - Bordado, Gabriel Jr. H.

- `b25a75ab-6c33-4b38-ae3e-276a569e8d57` - Brosas, Arlene D.

- `36a807b2-d73d-4681-baff-77993b8c00ea` - Calderon, Peter John D.

- `d40fe5dd-00f7-44fc-ad60-10df485c9345` - Campos, Luis Jose Angel Jr. N.

- `e07d2f56-3196-4a3a-9ee8-422757ebdafe` - Canama, Sabiniano S.

- `23012752-621c-4720-b34d-a0b70383ac71` - Casilao, Ariel 'Ka Ayik' B.

- `939907a9-cb71-439b-af89-34f383c1d64e` - Castro, France L.

- `6318741f-0964-4b89-a5b6-22cf71845b17` - Cayetano, Pia S.

- `c2da1c27-29e2-4800-b04f-2bd6950a257b` - Chavez, Cecilia Leonila V.

- `0d5f06fc-a834-434b-be4f-a32209d49d06` - Cojuangco, Carlos O.

- `daa6b670-3576-4f6f-be7b-eb316ed00c74` - Collantes, Ma. Theresa V.

- `695afaa7-b4eb-4d17-8b77-44edcef50dce` - Cortes, Jonas C.

- `41932dd6-55eb-4ea9-a16f-caae65217bf1` - Crisologo, Vincent "Bingbong" P.

- `c6a4e896-ad0e-4421-8ccc-426d24301999` - Cuaresma, Luisa Lloren

- `6a5df7cb-72f4-42d5-ad66-2bb4efbed992` - Dalipe, Manuel Jose "Mannix" M.

- `5d426fdb-931c-463a-a2cb-0402b02007ac` - De Venecia, Christopher "Toff" Vera Perez

- `4e252fab-9b4a-46a4-b7da-9edb9aeb51ba` - De Vera, Eugene Michael B.

- `2115df09-be10-4883-85ed-043d3ccac4e9` - Del Rosario, Manuel Monsour III T.

- `a6a024d4-2cb7-4cdc-90f5-22ce595d2b9c` - Dimaporo, Mohamad Khalid Q.

- `da728920-d855-46c7-b2fd-b98ce6174ca5` - Duavit, Michael John R.

- `8c296986-b303-4346-a95e-52dfebdd0f0a` - Durano, Ramon "Red" VI H.

- `e032e0c6-7624-4d8c-a91f-aae67e834f62` - Elago, Sarah Jane I.

- `cf0cc3b2-5b39-4c70-a0d1-fa6efa62279a` - Enverga, Anna Katrina "Trina" M.

- `63aa1d9c-184b-484b-b343-ef0ef64e6618` - Eriguel, Sandra Y., M.D.

- `c45b47d3-073a-4414-ae26-1b607fbcc53e` - Espino, Amado Jr. T.

- `5e6e0c08-74ee-41a2-87b3-f43e6abf6e46` - Eusebio, Richard C.

- `4d93de59-2968-49bf-bde8-ca26375e10d0` - Fernando, Bayani F.

- `3446c28f-3528-4572-a666-b77e7f88276e` - Ferrer, Juliet Marie D.

- `cd7e018f-fd81-499c-af99-fd6cdfdd4fcb` - Floirendo, Antonio Jr. R.

- `b589c80b-2138-43ff-a42c-fa21ea74ae7a` - Fuentebella, Arnulfo P.

- `8c834e3b-81ac-401d-9b3b-e92f7fca6ef8` - Garbin, Alfredo Jr. A.

- `33c4178a-a86b-4d8e-8db6-9b9a84451247` - Garcia, Jose Enrique III "Joet" S.

- `b6d3efba-5fab-4cef-aa76-86ab3f624d39` - Gasataya, Greg G.

- `4670ff20-a456-46c7-9214-c8833ae7d249` - Go, Mark O.

- `917ac704-6c45-4512-a9df-431e83ebe851` - Gonzaga, Ruwel Peter S.

- `1a00414c-9ced-4cc3-ac99-7945bd7bf9fb` - Gonzales, Alexandria P.

- `8fdc82d5-27b7-4aee-9de3-aa6d38e57f45` - Gonzales, Aurelio Jr. "Dong" D.

- `ec3f1dc0-9cfa-4241-9f06-8eff9f67b63d` - Herrera-Dy, Bernadette "BH"

- `b9d7f8d5-d0bd-4ca2-97f0-2bb47898af3c` - Labadlabad, Glona G.

- `e855d4a6-3e31-4809-bb75-110f3488a9e1` - Lacson, Virgilio S.

- `eda1bc3c-9879-494f-80ec-3a842172724e` - Lagman, Edcel C.

- `1abefbb4-f870-44e4-8e2d-8373763f9277` - Laogan, Dennis C.

- `42cb252b-fbd4-4cd7-9fee-f0590a68bec7` - Lazatin, Carmelo II "Jon" B.

- `f369ae98-9320-49f6-9f64-5f5fda2fe571` - Limkaichong, Jocelyn Sy

- `a86a40e3-6933-4d6c-b17f-42799c542433` - Lopez, Benhur Jr. B.

- `04600538-5894-42d9-891c-e3d5033fa3ef` - Lopez, Manuel Luis T.

- `935760e5-cb01-4972-97be-44e06d865fd1` - Maceda, Edward Vera Perez

- `74c0cd0f-cb1c-4491-b25f-662893d179d2` - Madrona, Emmanuel F.

- `ef71d31c-dbcb-41d0-b0cb-5784191d854d` - Malapitan, Dale "Along" R.

- `d7815e5b-dc5b-4859-bd6a-e2c9e3c779ce` - Mangaoang, Allen Jesse C.

- `16113a15-2bfb-4894-afc2-8f4d56a0402f` - Mangudadatu, Suharto T.

- `5184c195-b91d-47ec-b3c3-7e3afa401bae` - Marcoleta, Rodante D.

- `cc4cc4c3-6a6d-4fc8-8dbf-6b2e61bf726f` - Mariño, Mario Vittorio "Marvey" A.

- `c64dd58b-fa7c-49db-ab08-4272ea532b38` - Marquez, Carlito S.

- `ef5f0f9e-7f50-4105-916c-d2a7aab831d4` - Martinez, Eric M.

- `55d966d0-8b8c-4123-8592-80c6e9bbcd84` - Matugas, Francisco Jose II F.

- `bfd03f48-3887-46f8-942d-74b7f710230f` - Mending, Makmod Jr. D.

- `fec15344-6e83-429b-9c84-7c54dda21c8f` - Mercado, Roger G.

- `4aea1051-d04a-4cde-9102-f729ab4055e8` - Montoro, Teodoro "Ted" G.

- `3149dc74-f8b3-4e55-b0ec-e22a477b2a32` - Nava, Ma. Lucille L., M.D.

- `9240414c-ece2-4481-82cd-24097a4d6770` - Nieto, John Marvin "Yul Servo" C.

- `05362164-4790-48cd-9fd8-b5c774063486` - Nograles, Jericho Jonas B.

- `73528348-43f7-476b-af25-0330fe10d3eb` - Nolasco, Ramon C.

- `0d0a5126-fff9-4e2b-aae6-966266cffcee` - Nuñez-Malanyaon, Corazon N.

- `1fc06db8-5078-4444-89b6-8b126929f80a` - Ong, Edwin C.

- `8c55b381-ab96-4929-84b2-282c720d5750` - Ong, Henry C.

- `9bbe6de3-e2b3-486b-8a33-0ddb1c530c3a` - Ortega, Pablo C.

- `f2b3b2d5-0ddc-4479-b1f3-1452c082b01e` - Ortega, Vini Nola A.

- `a41586ce-cb02-443c-af30-09fb4bffeba3` - Pacquiao, Rogelio "Ruel" D.

- `2970a6fd-11ac-4395-88df-c135f938dc17` - Palma, Wilter "Sharky" II Wee

- `482543b1-3e26-4217-bb19-0d80de53feee` - Panotes, Marisol C.

- `76e5d070-6dfb-4f91-b459-b81a5960d618` - Papandayan, Mauyag Jr. "Jun" B.

- `27b482ce-aa94-4833-baab-b441bcd906e0` - Pichay, Prospero Jr. A.

- `7f9a2dcd-958a-487c-b942-adad38c7d13a` - Pimentel, Johnny Ty

- `1581bf0f-db71-41b7-bd3f-881420e62d72` - Pineda, Enrico A.

- `18a189e5-9978-42b2-98cb-c03dd555d4be` - Revilla, Strike B.

- `62b32ffb-a447-4d61-8745-513c534ae83c` - Roa-Puno, Cristina "Chiqui"

- `542f5f26-9685-4c13-b250-2bb2fa272d94` - Robes, Florida "Rida" P.

- `ab385fb4-f103-4bb5-b9f0-3aab6fd1ebd5` - Rocamora, Ramon V. A."Rav"

- `3de6f8b0-32ae-48e7-85ac-8c13485ba82e` - Roman, Geraldine B.

- `0a347cd0-8a8a-494f-bd26-250ea9c2c17f` - Romero, Michael L., PhD.

- `39c9e954-f32f-4bf4-87b2-0cebb430cc17` - Romualdez, Yedda Marie K.

- `0b742604-c895-49a7-983c-5d14111eeb84` - Roque, Harry Jr. L.

- `7adff77b-fcb2-4f60-b6e3-5926824b41d9` - Sagarbarria, Manuel T.

- `e88243d9-ca4b-4566-bbc0-4312a4773653` - Salceda, Joey Sarte

- `e6d71ca7-3523-421c-9e1b-b78371279e8d` - Salo, Ron P.

- `2bbe94d3-14d8-4da6-a61b-d005410261e2` - Salon, Orestes T.

- `3f4f4355-94b6-4d3b-803e-0fd2da2ae496` - Sambar, Mark Aeron H.

- `f2fb1d38-10a1-4f4c-b48b-59f397661142` - Sandoval, Federico II "Ricky" S.

- `0563b0c0-7a75-4809-b43e-6f26bb1f829b` - Santos-Recto, Vilma

- `59614a31-6159-438e-893a-2e904ebd75f3` - Sarmiento, Edgar Mary S.

- `1fbe5488-18ab-41aa-95f0-3fa38ae3d768` - Savellano, Deogracias Victor "DV" B.

- `64fae2cb-920a-4c88-9116-c43d0c9d3f9a` - Siao, Frederick W.

- `ec838ab6-ef30-46dc-8f50-487a62c6d1d5` - Silverio, Lorna C.

- `3a13887f-cbb4-43d0-bc6a-fc642f419ba5` - Suansing, Horacio Jr. P.

- `ab0ed65d-9eef-48a5-bfdb-241f7c4d0482` - Suarez, Danilo E.

- `c45a526d-1034-4e4b-bcc2-161d7830f9d9` - Sy-Alvarado, Jose Antonio "Kuya Jonathan" R.

- `6ba651a8-b183-47d1-82a8-ee32b178b764` - Tan, Shernee Abubakar

- `a4814cdb-a0b4-4199-a53d-11af4ef96dfb` - Teves, Arnolfo Jr. "Arnie" A.

- `fba86ae8-06fc-40f0-90bc-2ea9844e508a` - Tupas, Raul "Boboy" C.

- `86212270-47aa-4f5c-a7e6-605c1781e7e0` - Ungab, Alberto T.

- `5b6c1179-df36-488e-b45e-714f15bde8e5` - Unico, Renato Jr. J.

- `2d3b686f-1075-4812-849c-e6b4a172d18d` - Uybarreta, Carlos Roman

- `e442d9e4-4f23-4361-912c-b9f191c375e0` - Velasco-Catera, Tricia Nicole Q.

- `af5da708-67fd-45e9-84fb-fbd5518f8839` - Veloso, Vicente "Ching" S.E.

- `2a0cd500-1337-4736-8fb2-cad2ef1426c7` - Vergara, Rosanna "Ria" V.

- `114599ed-b43e-49ba-99a2-3034c2999561` - Villafuerte, Luis Raymund Jr. F.

- `75965cd5-b5a4-4630-903b-55580affb608` - Villaraza-Suarez, Anna Marie

- `48868a3b-1a21-4063-b037-d2c3bbce347b` - Villarin, Tom S.

- `0b5c5754-a1f5-42a4-a412-620e388eb9a9` - Violago, Micaela S.

- `00a98373-944f-4def-9e9e-08ac1f36314e` - Yap, Melecio Jr. J.

- `d298743a-8d68-4d10-8c8f-fc50d2ca8931` - Yap, Victor A.

- `b1b798b1-31a8-41bd-9bfd-73f826530c78` - Yu, Divina Grace C.

- `99228975-a9f4-4bf3-8bb0-353ba76e905b` - Zubiri, Manuel F.



### Removed


No people removed


### Name Changes


No name changes


### Additional Name Changes


No name changes


### Wikidata Changes


No changes


## Organizations

### Added



- `party/1-ang_edukasyon` - 1-ANG EDUKASYON

- `party/1-care` - 1-CARE

- `party/1-pacman` - 1-PACMAN

- `party/aangat_tayo` - AANGAT TAYO

- `party/aasenso` - AASENSO

- `party/acts_ofw` - ACTS OFW

- `party/alona` - ALONA

- `party/ang_kabuhayan` - ANG KABUHAYAN

- `party/bh` - BH

- `party/kabayan` - KABAYAN

- `party/kusug-tausug` - KUSUG-TAUSUG

- `party/manila_teachers` - MANILA TEACHERS

- `party/mata` - MATA

- `party/pba` - PBA

- `party/sagip` - SAGIP

- `party/sbp` - SBP



### Removed


No organizations removed


## Memberships

### Added


#### term/17

- Abad, Henedina R. 

- Abaya, Francis Gerald A. 

- Abayon, Harlin Neil III J. 

- Abellanosa, Rodrigo A. 

- Abu, Raneo E. 

- Abueg, Frederick "Erick" F. 

- Acharon, Pedro Jr. B. 

- Acop, Romeo M. 

- Acosta, Gil "Kabarangay" P. 

- Acosta-Alba, Ma. Lourdes 

- Adiong, Ansaruddin A.M. A. 

- Advincula, Alex "AA" L. 

- Agarao, Benjamin Jr. C. 

- Aggabao, Ma. Lourdes R. 

- Aglipay Villar, Emmeline Y. 

- Akbar, Jum J. 

- Albano, Rodolfo III T. 

- Alcala, Vicente "Kulit" J. 

- Alejano, Gary C. 

- Almario, Joel Mayo Z. 

- Almonte, Jorge T. 

- Alonte-Naguiat, Len B. 

- Alvarez, Franz "Chicoy" E. 

- Alvarez, Mercedes K. 

- Alvarez, Pantaleon D. 

- Amante, Erlpe John "Ping" M. 

- Amatong, Isagani S. 

- Andaya, Rolando Jr. G. 

- Angara-Castillo, Bellaflor J. 

- Antonino, Magnolia Rosa C. 

- Antonio, Michelle M. 

- Aragones, Sol 

- Arbison, Munir M. 

- Arcillas, Arlene B. 

- Arenas, Rose Marie "Baby" J. 

- Atienza, Jose Jr. L. 

- Aumentado, Erico Aristotle C. 

- Austria Barzaga, Jennifer 

- Bag-ao, Kaka J. 

- Bagatsing, Cristal L. 

- Baguilat, Teddy Jr. Brawner 

- Banal, Jorge"Bolet" B. 

- Barbers, Robert Ace S. 

- Bataoil, Leopoldo N. 

- Batocabe, Rodel M. 

- Bautista-Bandigan, Lorna P. 

- Belaro, Salvador Jr. B. 

- Belmonte, Feliciano Jr. R. 

- Belmonte, Jose Christopher Y. 

- Belmonte, Ricardo Jr. T. 

- Benitez, Alfredo "Albee" B. 

- Bernos, Joseph Sto. Niño B. 

- Bertiz, Aniceto "John" III D. 

- Biazon, Rozzano Rufino B. 

- Billones, Emmanuel A. 

- Biron, Ferjenel G., M.D. 

- Bolilia, Lianda B. 

- Bondoc, Juan Pablo "Rimpy" P. 

- Bordado, Gabriel Jr. H. 

- Bravo, Anthony M., Ph.D. 

- Bravo, Maria Vida E. 

- Brosas, Arlene D. 

- Bulut-Begtang, Eleanor C. 

- Cagas, Mercedes C. 

- Calderon, Peter John D. 

- Calixto-Rubiano, Emi G. 

- Caminero, Wilfredo S. 

- Campos, Luis Jose Angel Jr. N. 

- Canama, Sabiniano S. 

- Cari, Jose Carlos L. 

- Casilao, Ariel 'Ka Ayik' B. 

- Castelo, Winston "Winnie" 

- Castro, France L. 

- Castro, Fredenil H. 

- Catamco, Nancy A. 

- Cayetano, Pia S. 

- Celeste, Jesus "Boying" F. 

- Cerafica, Arnel M. 

- Cerilles, Aurora Enerio 

- Chavez, Cecilia Leonila V. 

- Chipeco, Joaquin Jr. M. 

- Co, Christopher S. 

- Cojuangco, Carlos O. 

- Collantes, Ma. Theresa V. 

- Cortes, Jonas C. 

- Cortuna, Julieta R. 

- Cosalan, Ronald M. 

- Crisologo, Vincent "Bingbong" P. 

- Cua, Dakila Carlo E. 

- Cuaresma, Luisa Lloren 

- Cueva, Leo Rafael M. 

- Dalipe, Manuel Jose "Mannix" M. 

- Dalog, Maximo B. 

- Daza, Raul A. 

- De Jesus, Emmi A. 

- De Venecia, Christopher "Toff" Vera Perez 

- De Vera, Eugene Michael B. 

- Defensor, Arthur Jr. R. 

- Del Mar, Raul V. 

- Del Rosario, Manuel Monsour III T. 

- Deloso-Montalla, Cheryl P. 

- Dimaporo, Abdullah D. 

- Dimaporo, Mohamad Khalid Q. 

- Duavit, Michael John R. 

- Durano, Ramon "Red" VI H. 

- Dy, Napoleon S. 

- Elago, Sarah Jane I. 

- Enverga, Anna Katrina "Trina" M. 

- Erice, Edgar R. 

- Eriguel, Sandra Y., M.D. 

- Ermita-Buhain, Elenita Milagros "Eileen" 

- Escudero, Evelina G. 

- Espina, Rogelio J.,M.D. 

- Espino, Amado Jr. T. 

- Estrella, Conrado III M. 

- Eusebio, Richard C. 

- Evardone, Ben P. 

- Fariñas, Rodolfo C. 

- Fernando, Bayani F. 

- Ferrer, Juliet Marie D. 

- Ferrer, Luis "Jon-Jon" IV A. 

- Ferriol-Pascual, Abigail Faye C. 

- Floirendo, Antonio Jr. R. 

- Flores, Florencio Jr. T., M.D. 

- Fortun, Lawrence Lemuel H. 

- Fortuno, Salvio B. 

- Fuentebella, Arnulfo P. 

- Garbin, Alfredo Jr. A. 

- Garcia, Gwendolyn F. 

- Garcia, Jose Enrique III "Joet" S. 

- Garcia-Albano, Mylene J. 

- Garin, Oscar "Richard" Jr. S. 

- Garin, Sharon S. 

- Gasataya, Greg G. 

- Gatchalian, Wes 

- Geron, Rico B. 

- Go, Ana Cristina Siquian 

- Go, Mark O. 

- Gomez, Lucy T. 

- Gonzaga, Ruwel Peter S. 

- Gonzales, Alexandria P. 

- Gonzales, Aurelio Jr. "Dong" D. 

- Gonzalez, Fernando V. 

- Gorriceta, Arcadio H. 

- Gullas, Gerald Anthony Jr. V. 

- Hernandez, Ferdinand L. 

- Herrera-Dy, Bernadette "BH" 

- Hofer, Ann K. 

- Jalosjos, Seth Frederick P. 

- Javier, Paolo S. 

- Kho, Elisa "Olga" T. 

- Khonghun, Jeffrey D. 

- Labadlabad, Glona G. 

- Lacson, Virgilio S. 

- Lagman, Edcel C. 

- Lanete, Scott Davies S., M.D. 

- Laogan, Dennis C. 

- Lazatin, Carmelo II "Jon" B. 

- Leachon, Doy C. 

- Lee, Delphine "Delph" Gan 

- Limkaichong, Jocelyn Sy 

- Lobregat, Celso L. 

- Loong, Tupay T. 

- Lopez, Benhur Jr. B. 

- Lopez, Carlo V. 

- Lopez, Manuel Luis T. 

- Loyola, Roy M. 

- Macapagal-Arroyo, Gloria 

- Maceda, Edward Vera Perez 

- Madrona, Emmanuel F. 

- Malapitan, Dale "Along" R. 

- Manalo, Jesulito A. 

- Mangaoang, Allen Jesse C. 

- Mangudadatu, Suharto T. 

- Mangudadatu, Zajid G. 

- Marcoleta, Rodante D. 

- Marcos, Imelda R. 

- Mariño, Mario Vittorio "Marvey" A. 

- Marquez, Carlito S. 

- Martinez, Eric M. 

- Matugas, Francisco Jose II F. 

- Mellana, Evelyn P. 

- Mending, Makmod Jr. D. 

- Mercado, Roger G. 

- Mirasol, Alejandro Y. 

- Montoro, Teodoro "Ted" G. 

- Nava, Ma. Lucille L., M.D. 

- Nieto, John Marvin "Yul Servo" C. 

- Noel, Victoria Isabel G. 

- Nograles, Jericho Jonas B. 

- Nograles, Karlo Alexei B. 

- Nolasco, Ramon C. 

- Nuñez-Malanyaon, Corazon N. 

- Oaminal, Henry S. 

- Ocampo, Rosenda Ann 

- Olivarez, Eric L. 

- Ong, Edwin C. 

- Ong, Henry C. 

- Ortega, Pablo C. 

- Ortega, Vini Nola A. 

- Pacquiao, Rogelio "Ruel" D. 

- Paduano, Joseph Stephen S. 

- Palma, Wilter "Sharky" II Wee 

- Pancho, Gavini C. 

- Panganiban, Jose Jr. T. 

- Panotes, Marisol C. 

- Papandayan, Mauyag Jr. "Jun" B. 

- Pichay, Prospero Jr. A. 

- Pimentel, Johnny Ty 

- Pineda, Enrico A. 

- Plaza, Maria Valentina G. 

- Primicias-Agabas, Marlyn L. 

- Quimbo, Romero "Miro" S. 

- Radaza, Aileen C. 

- Ramos, Deogracias Jr. B. 

- Relampagos, Rene L. 

- Revilla, Strike B. 

- Roa-Puno, Cristina "Chiqui" 

- Robes, Florida "Rida" P. 

- Rocamora, Ramon V. A."Rav" 

- Rodriguez, Isidro Jr. S. 

- Rodriguez, Maximo Jr. B. 

- Roman, Geraldine B. 

- Romero, Michael L., PhD. 

- Romualdez, Yedda Marie K. 

- Romualdo, Xavier Jesus "XJ" D. 

- Roque, Harry Jr. L. 

- Roque, Rogelio Neil Pepito 

- Sacdalan, Jesus N. 

- Sagarbarria, Manuel T. 

- Sahali, Ruby M. 

- Salceda, Joey Sarte 

- Salimbangon, Benhur L. 

- Salo, Ron P. 

- Salon, Orestes T. 

- Sambar, Mark Aeron H. 

- Sandoval, Federico II "Ricky" S. 

- Santos-Recto, Vilma 

- Sarmiento, Cesar V. 

- Sarmiento, Edgar Mary S. 

- Sato, Josephine R. 

- Savellano, Deogracias Victor "DV" B. 

- Sema, Bai Sandra A. 

- Siao, Frederick W. 

- Silverio, Lorna C. 

- Singson, Eric D. 

- Suansing, Estrellita B. 

- Suansing, Horacio Jr. P. 

- Suarez, Danilo E. 

- Sy-Alvarado, Jose Antonio "Kuya Jonathan" R. 

- Tambunting, Gus S. 

- Tan, Angelina "Helen" DL, M.D. 

- Tan, Milagrosa T. 

- Tan, Shernee Abubakar 

- Tejada, Jose I. 

- Teves, Arnolfo Jr. "Arnie" A. 

- Tiangco, Tobias "Toby" M. 

- Ting, Randolph S. 

- Tinio, Antonio L. 

- Tolentino, Abraham "Bambol" N. 

- Treñas, Jerry P. 

- Tugna, Sherwin N. 

- Tupas, Raul "Boboy" C. 

- Turabin-Hataman, Sitti Djalia A. 

- Ty, Arnel U. 

- Umali, Reynaldo V. 

- Unabia, Peter "Sr. Pedro" M. 

- Ungab, Alberto T. 

- Unico, Renato Jr. J. 

- Uy, Juliette T. 

- Uy, Rolando "Klarex" A. 

- Uybarreta, Carlos Roman 

- Vargas, Alfredo III D. 

- Vargas-Alfonso, Baby Aline 

- Velarde, Mariano Michael Jr. M. 

- Velasco, Lord Allan Jay Q. 

- Velasco-Catera, Tricia Nicole Q. 

- Veloso, Vicente "Ching" S.E. 

- Vergara, Rosanna "Ria" V. 

- Villafuerte, Luis Raymund Jr. F. 

- Villanueva, Noel L. 

- Villar, Mark A. 

- Villaraza-Suarez, Anna Marie 

- Villarica, Linabelle Ruth R. 

- Villarin, Tom S. 

- Violago, Micaela S. 

- Yap, Arthur C. 

- Yap, Melecio Jr. J. 

- Yap, Victor A. 

- Yu, Divina Grace C. 

- Zamora, Maria Carmen S. 

- Zamora, Ronaldo B. 

- Zarate, Carlos Isagani T. 

- Zubiri, Manuel F. 




### Removed

No memberships removed


## Terms

### Added



- `term/17` - 17th Congress of the Philippines



### Removed


No terms removed


## Elections

### Added


No elections added


### Removed


No elections removed

tmtmtmtm commented on data/Philippines/House/sources/instructions.json:
```diff
@@ -6,6 +6,21 @@
+      "merge": {
+        "incoming_field": "name",
+        "existing_field": "name",
+        "reconciliation_file": "reconciliation/data.csv"
```
You probably want to explain in the commit message why you're matching by `name` here, rather than by `id`

tmtmtmtm commented:
@octopusinvitro Your remote checklist (the relevant parts of which will need to be added to this issue) currently has a tick against 'wikidata is being reconciled'. However, you're currently only reconciling against the Wikidata for the members of the prior term, not the members of this term. 

So I [updated](https://github.com/everypolitician-scrapers/philippines-house-of-representatives-wikidata/commit/9ea7d7fcb4def707ee68a7b017d5d49dc40035ec) the scraper to include the new members, and have re-run it. You'll now need to re-reconcile.

octopusinvitro commented:
Summary of changes in `data/Philippines/House/ep-popolo-v1.0.json`:

## People

### Added



- `62717031-beb0-44b8-b894-37777c617ed4` - Abayon, Harlin Neil III J.

- `acc2853f-7e76-49fb-8184-5e7282bcc7dc` - Acosta, Gil "Kabarangay" P.

- `226421ae-f958-4289-8a8b-22e3a5f57cc1` - Aggabao, Ma. Lourdes R.

- `0082ce21-03e7-4f0b-b837-0229590f6fb2` - Akbar, Jum J.

- `a88eb0c4-bc8f-432e-9645-e04c868b3b66` - Almario, Joel Mayo Z.

- `373a9abb-aed4-461b-a42f-ea2b34103c58` - Alonte-Naguiat, Len B.

- `7a820239-e3a9-4b21-8fb1-e8cbba7c2b7e` - Alvarez, Pantaleon D.

- `e012ef3c-0acd-4d88-b274-c93dda9e6a76` - Antonio, Michelle M.

- `abd7a941-a106-494c-8244-0fdb9cab77f2` - Arbison, Munir M.

- `9fe58df4-f6f2-4aea-8b5a-da89fba4dfff` - Arcillas, Arlene B.

- `285a8439-3477-44b4-896b-945c36781b61` - Austria Barzaga, Jennifer

- `7b3162b2-0d2e-46db-ba97-22af8f43fc0d` - Bagatsing, Cristal L.

- `7ee62ce0-3450-40e0-aabe-1211516c824b` - Barbers, Robert Ace S.

- `d77a2ee9-859c-41ba-853c-725836ef876f` - Bautista-Bandigan, Lorna P.

- `60dce242-08c5-4b6a-ae52-662d8d9c9cb7` - Belaro, Salvador Jr. B.

- `8f97c112-56f0-4690-be47-206c611cf39b` - Belmonte, Ricardo Jr. T.

- `9f2c1150-7553-44b9-afcd-a9065384cc79` - Bernos, Joseph Sto. Niño B.

- `4356189b-3575-4c5f-bb11-2d662886a6ed` - Bertiz, Aniceto "John" III D.

- `eefc4b51-3986-4817-a02f-b3fe0054a940` - Biazon, Rozzano Rufino B.

- `ab1cdda3-b74d-4246-a726-4623776eb7e7` - Billones, Emmanuel A.

- `7cd865f0-8b4f-464c-bcc2-a21240bf4144` - Biron, Ferjenel G., M.D.

- `7ab80a30-8dc5-44b6-b4df-24cdab6cc883` - Bolilia, Lianda B.

- `c5560722-9c7c-45d4-bc0a-aec7039efb7a` - Bordado, Gabriel Jr. H.

- `b25a75ab-6c33-4b38-ae3e-276a569e8d57` - Brosas, Arlene D.

- `36a807b2-d73d-4681-baff-77993b8c00ea` - Calderon, Peter John D.

- `d40fe5dd-00f7-44fc-ad60-10df485c9345` - Campos, Luis Jose Angel Jr. N.

- `e07d2f56-3196-4a3a-9ee8-422757ebdafe` - Canama, Sabiniano S.

- `23012752-621c-4720-b34d-a0b70383ac71` - Casilao, Ariel 'Ka Ayik' B.

- `939907a9-cb71-439b-af89-34f383c1d64e` - Castro, France L.

- `6318741f-0964-4b89-a5b6-22cf71845b17` - Cayetano, Pia S.

- `c2da1c27-29e2-4800-b04f-2bd6950a257b` - Chavez, Cecilia Leonila V.

- `0d5f06fc-a834-434b-be4f-a32209d49d06` - Cojuangco, Carlos O.

- `daa6b670-3576-4f6f-be7b-eb316ed00c74` - Collantes, Ma. Theresa V.

- `695afaa7-b4eb-4d17-8b77-44edcef50dce` - Cortes, Jonas C.

- `41932dd6-55eb-4ea9-a16f-caae65217bf1` - Crisologo, Vincent "Bingbong" P.

- `c6a4e896-ad0e-4421-8ccc-426d24301999` - Cuaresma, Luisa Lloren

- `6a5df7cb-72f4-42d5-ad66-2bb4efbed992` - Dalipe, Manuel Jose "Mannix" M.

- `5d426fdb-931c-463a-a2cb-0402b02007ac` - De Venecia, Christopher "Toff" Vera Perez

- `4e252fab-9b4a-46a4-b7da-9edb9aeb51ba` - De Vera, Eugene Michael B.

- `2115df09-be10-4883-85ed-043d3ccac4e9` - Del Rosario, Manuel Monsour III T.

- `a6a024d4-2cb7-4cdc-90f5-22ce595d2b9c` - Dimaporo, Mohamad Khalid Q.

- `da728920-d855-46c7-b2fd-b98ce6174ca5` - Duavit, Michael John R.

- `8c296986-b303-4346-a95e-52dfebdd0f0a` - Durano, Ramon "Red" VI H.

- `e032e0c6-7624-4d8c-a91f-aae67e834f62` - Elago, Sarah Jane I.

- `cf0cc3b2-5b39-4c70-a0d1-fa6efa62279a` - Enverga, Anna Katrina "Trina" M.

- `63aa1d9c-184b-484b-b343-ef0ef64e6618` - Eriguel, Sandra Y., M.D.

- `c45b47d3-073a-4414-ae26-1b607fbcc53e` - Espino, Amado Jr. T.

- `5e6e0c08-74ee-41a2-87b3-f43e6abf6e46` - Eusebio, Richard C.

- `4d93de59-2968-49bf-bde8-ca26375e10d0` - Fernando, Bayani F.

- `3446c28f-3528-4572-a666-b77e7f88276e` - Ferrer, Juliet Marie D.

- `cd7e018f-fd81-499c-af99-fd6cdfdd4fcb` - Floirendo, Antonio Jr. R.

- `b589c80b-2138-43ff-a42c-fa21ea74ae7a` - Fuentebella, Arnulfo P.

- `8c834e3b-81ac-401d-9b3b-e92f7fca6ef8` - Garbin, Alfredo Jr. A.

- `33c4178a-a86b-4d8e-8db6-9b9a84451247` - Garcia, Jose Enrique III "Joet" S.

- `b6d3efba-5fab-4cef-aa76-86ab3f624d39` - Gasataya, Greg G.

- `4670ff20-a456-46c7-9214-c8833ae7d249` - Go, Mark O.

- `917ac704-6c45-4512-a9df-431e83ebe851` - Gonzaga, Ruwel Peter S.

- `1a00414c-9ced-4cc3-ac99-7945bd7bf9fb` - Gonzales, Alexandria P.

- `8fdc82d5-27b7-4aee-9de3-aa6d38e57f45` - Gonzales, Aurelio Jr. "Dong" D.

- `ec3f1dc0-9cfa-4241-9f06-8eff9f67b63d` - Herrera-Dy, Bernadette "BH"

- `b9d7f8d5-d0bd-4ca2-97f0-2bb47898af3c` - Labadlabad, Glona G.

- `e855d4a6-3e31-4809-bb75-110f3488a9e1` - Lacson, Virgilio S.

- `eda1bc3c-9879-494f-80ec-3a842172724e` - Lagman, Edcel C.

- `1abefbb4-f870-44e4-8e2d-8373763f9277` - Laogan, Dennis C.

- `42cb252b-fbd4-4cd7-9fee-f0590a68bec7` - Lazatin, Carmelo II "Jon" B.

- `f369ae98-9320-49f6-9f64-5f5fda2fe571` - Limkaichong, Jocelyn Sy

- `a86a40e3-6933-4d6c-b17f-42799c542433` - Lopez, Benhur Jr. B.

- `04600538-5894-42d9-891c-e3d5033fa3ef` - Lopez, Manuel Luis T.

- `935760e5-cb01-4972-97be-44e06d865fd1` - Maceda, Edward Vera Perez

- `74c0cd0f-cb1c-4491-b25f-662893d179d2` - Madrona, Emmanuel F.

- `ef71d31c-dbcb-41d0-b0cb-5784191d854d` - Malapitan, Dale "Along" R.

- `d7815e5b-dc5b-4859-bd6a-e2c9e3c779ce` - Mangaoang, Allen Jesse C.

- `16113a15-2bfb-4894-afc2-8f4d56a0402f` - Mangudadatu, Suharto T.

- `5184c195-b91d-47ec-b3c3-7e3afa401bae` - Marcoleta, Rodante D.

- `cc4cc4c3-6a6d-4fc8-8dbf-6b2e61bf726f` - Mariño, Mario Vittorio "Marvey" A.

- `c64dd58b-fa7c-49db-ab08-4272ea532b38` - Marquez, Carlito S.

- `ef5f0f9e-7f50-4105-916c-d2a7aab831d4` - Martinez, Eric M.

- `55d966d0-8b8c-4123-8592-80c6e9bbcd84` - Matugas, Francisco Jose II F.

- `bfd03f48-3887-46f8-942d-74b7f710230f` - Mending, Makmod Jr. D.

- `fec15344-6e83-429b-9c84-7c54dda21c8f` - Mercado, Roger G.

- `4aea1051-d04a-4cde-9102-f729ab4055e8` - Montoro, Teodoro "Ted" G.

- `3149dc74-f8b3-4e55-b0ec-e22a477b2a32` - Nava, Ma. Lucille L., M.D.

- `9240414c-ece2-4481-82cd-24097a4d6770` - Nieto, John Marvin "Yul Servo" C.

- `05362164-4790-48cd-9fd8-b5c774063486` - Nograles, Jericho Jonas B.

- `73528348-43f7-476b-af25-0330fe10d3eb` - Nolasco, Ramon C.

- `0d0a5126-fff9-4e2b-aae6-966266cffcee` - Nuñez-Malanyaon, Corazon N.

- `1fc06db8-5078-4444-89b6-8b126929f80a` - Ong, Edwin C.

- `8c55b381-ab96-4929-84b2-282c720d5750` - Ong, Henry C.

- `9bbe6de3-e2b3-486b-8a33-0ddb1c530c3a` - Ortega, Pablo C.

- `f2b3b2d5-0ddc-4479-b1f3-1452c082b01e` - Ortega, Vini Nola A.

- `a41586ce-cb02-443c-af30-09fb4bffeba3` - Pacquiao, Rogelio "Ruel" D.

- `2970a6fd-11ac-4395-88df-c135f938dc17` - Palma, Wilter "Sharky" II Wee

- `482543b1-3e26-4217-bb19-0d80de53feee` - Panotes, Marisol C.

- `76e5d070-6dfb-4f91-b459-b81a5960d618` - Papandayan, Mauyag Jr. "Jun" B.

- `27b482ce-aa94-4833-baab-b441bcd906e0` - Pichay, Prospero Jr. A.

- `7f9a2dcd-958a-487c-b942-adad38c7d13a` - Pimentel, Johnny Ty

- `1581bf0f-db71-41b7-bd3f-881420e62d72` - Pineda, Enrico A.

- `18a189e5-9978-42b2-98cb-c03dd555d4be` - Revilla, Strike B.

- `62b32ffb-a447-4d61-8745-513c534ae83c` - Roa-Puno, Cristina "Chiqui"

- `542f5f26-9685-4c13-b250-2bb2fa272d94` - Robes, Florida "Rida" P.

- `ab385fb4-f103-4bb5-b9f0-3aab6fd1ebd5` - Rocamora, Ramon V. A."Rav"

- `3de6f8b0-32ae-48e7-85ac-8c13485ba82e` - Roman, Geraldine B.

- `0a347cd0-8a8a-494f-bd26-250ea9c2c17f` - Romero, Michael L., PhD.

- `39c9e954-f32f-4bf4-87b2-0cebb430cc17` - Romualdez, Yedda Marie K.

- `0b742604-c895-49a7-983c-5d14111eeb84` - Roque, Harry Jr. L.

- `7adff77b-fcb2-4f60-b6e3-5926824b41d9` - Sagarbarria, Manuel T.

- `e88243d9-ca4b-4566-bbc0-4312a4773653` - Salceda, Joey Sarte

- `e6d71ca7-3523-421c-9e1b-b78371279e8d` - Salo, Ron P.

- `2bbe94d3-14d8-4da6-a61b-d005410261e2` - Salon, Orestes T.

- `3f4f4355-94b6-4d3b-803e-0fd2da2ae496` - Sambar, Mark Aeron H.

- `f2fb1d38-10a1-4f4c-b48b-59f397661142` - Sandoval, Federico II "Ricky" S.

- `0563b0c0-7a75-4809-b43e-6f26bb1f829b` - Santos-Recto, Vilma

- `59614a31-6159-438e-893a-2e904ebd75f3` - Sarmiento, Edgar Mary S.

- `1fbe5488-18ab-41aa-95f0-3fa38ae3d768` - Savellano, Deogracias Victor "DV" B.

- `64fae2cb-920a-4c88-9116-c43d0c9d3f9a` - Siao, Frederick W.

- `ec838ab6-ef30-46dc-8f50-487a62c6d1d5` - Silverio, Lorna C.

- `3a13887f-cbb4-43d0-bc6a-fc642f419ba5` - Suansing, Horacio Jr. P.

- `ab0ed65d-9eef-48a5-bfdb-241f7c4d0482` - Suarez, Danilo E.

- `c45a526d-1034-4e4b-bcc2-161d7830f9d9` - Sy-Alvarado, Jose Antonio "Kuya Jonathan" R.

- `6ba651a8-b183-47d1-82a8-ee32b178b764` - Tan, Shernee Abubakar

- `a4814cdb-a0b4-4199-a53d-11af4ef96dfb` - Teves, Arnolfo Jr. "Arnie" A.

- `fba86ae8-06fc-40f0-90bc-2ea9844e508a` - Tupas, Raul "Boboy" C.

- `86212270-47aa-4f5c-a7e6-605c1781e7e0` - Ungab, Alberto T.

- `5b6c1179-df36-488e-b45e-714f15bde8e5` - Unico, Renato Jr. J.

- `2d3b686f-1075-4812-849c-e6b4a172d18d` - Uybarreta, Carlos Roman

- `e442d9e4-4f23-4361-912c-b9f191c375e0` - Velasco-Catera, Tricia Nicole Q.

- `af5da708-67fd-45e9-84fb-fbd5518f8839` - Veloso, Vicente "Ching" S.E.

- `2a0cd500-1337-4736-8fb2-cad2ef1426c7` - Vergara, Rosanna "Ria" V.

- `114599ed-b43e-49ba-99a2-3034c2999561` - Villafuerte, Luis Raymund Jr. F.

- `75965cd5-b5a4-4630-903b-55580affb608` - Villaraza-Suarez, Anna Marie

- `48868a3b-1a21-4063-b037-d2c3bbce347b` - Villarin, Tom S.

- `0b5c5754-a1f5-42a4-a412-620e388eb9a9` - Violago, Micaela S.

- `00a98373-944f-4def-9e9e-08ac1f36314e` - Yap, Melecio Jr. J.

- `d298743a-8d68-4d10-8c8f-fc50d2ca8931` - Yap, Victor A.

- `b1b798b1-31a8-41bd-9bfd-73f826530c78` - Yu, Divina Grace C.

- `99228975-a9f4-4bf3-8bb0-353ba76e905b` - Zubiri, Manuel F.



### Removed


No people removed


### Name Changes


No name changes


### Additional Name Changes



- `e2520057-7792-468f-a56f-b4b0bdab2ed4` (Andaya, Rolando Jr. G.): Added: Rolando Andaya, Jr..

- `899c9ac7-1a5c-4fa0-a3ab-1cfbc43d2bb9` (Baguilat, Teddy Jr. Brawner): Added: Teddy Baguilat ﹠ Teddy Baguilat jr..

- `85616864-e3ac-4f1c-b7c3-a38e62aef718` (Bello, Walden F.): Added: 瓦爾登·貝約.

- `5da3d559-95c0-40ae-b71e-8c9f728f81a4` (Cerilles, Aurora Enerio): Added: Aurora Enerio-Cerilles.

- `97208c2c-cea2-415d-80b7-a6ac4bb96390` (Kho, Elisa "Olga" T.): Added: Elisa Olga Kho.

- `d1b50722-4eab-4476-9c05-3203d1bbd899` (Lagman, Edcel "Grex" B.): Added: Edcel Lagman.



### Wikidata Changes



- `e2520057-7792-468f-a56f-b4b0bdab2ed4`: none → Q2163117

- `899c9ac7-1a5c-4fa0-a3ab-1cfbc43d2bb9`: none → Q2194971

- `5da3d559-95c0-40ae-b71e-8c9f728f81a4`: none → Q4822523

- `97208c2c-cea2-415d-80b7-a6ac4bb96390`: none → Q5361378

- `d1b50722-4eab-4476-9c05-3203d1bbd899`: none → Q5335744



## Organizations

### Added



- `party/1-ang_edukasyon` - 1-ANG EDUKASYON

- `party/1-care` - 1-CARE

- `party/1-pacman` - 1-PACMAN

- `party/aangat_tayo` - AANGAT TAYO

- `party/aasenso` - AASENSO

- `party/acts_ofw` - ACTS OFW

- `party/alona` - ALONA

- `party/ang_kabuhayan` - ANG KABUHAYAN

- `party/bh` - BH

- `party/kabayan` - KABAYAN

- `party/kusug-tausug` - KUSUG-TAUSUG

- `party/manila_teachers` - MANILA TEACHERS

- `party/mata` - MATA

- `party/pba` - PBA

- `party/sagip` - SAGIP

- `party/sbp` - SBP



### Removed


No organizations removed


## Memberships

### Added


#### term/17

- Abad, Henedina R. 

- Abaya, Francis Gerald A. 

- Abayon, Harlin Neil III J. 

- Abellanosa, Rodrigo A. 

- Abu, Raneo E. 

- Abueg, Frederick "Erick" F. 

- Acharon, Pedro Jr. B. 

- Acop, Romeo M. 

- Acosta, Gil "Kabarangay" P. 

- Acosta-Alba, Ma. Lourdes 

- Adiong, Ansaruddin A.M. A. 

- Advincula, Alex "AA" L. 

- Agarao, Benjamin Jr. C. 

- Aggabao, Ma. Lourdes R. 

- Aglipay Villar, Emmeline Y. 

- Akbar, Jum J. 

- Albano, Rodolfo III T. 

- Alcala, Vicente "Kulit" J. 

- Alejano, Gary C. 

- Almario, Joel Mayo Z. 

- Almonte, Jorge T. 

- Alonte-Naguiat, Len B. 

- Alvarez, Franz "Chicoy" E. 

- Alvarez, Mercedes K. 

- Alvarez, Pantaleon D. 

- Amante, Erlpe John "Ping" M. 

- Amatong, Isagani S. 

- Andaya, Rolando Jr. G. 

- Angara-Castillo, Bellaflor J. 

- Antonino, Magnolia Rosa C. 

- Antonio, Michelle M. 

- Aragones, Sol 

- Arbison, Munir M. 

- Arcillas, Arlene B. 

- Arenas, Rose Marie "Baby" J. 

- Atienza, Jose Jr. L. 

- Aumentado, Erico Aristotle C. 

- Austria Barzaga, Jennifer 

- Bag-ao, Kaka J. 

- Bagatsing, Cristal L. 

- Baguilat, Teddy Jr. Brawner 

- Banal, Jorge"Bolet" B. 

- Barbers, Robert Ace S. 

- Bataoil, Leopoldo N. 

- Batocabe, Rodel M. 

- Bautista-Bandigan, Lorna P. 

- Belaro, Salvador Jr. B. 

- Belmonte, Feliciano Jr. R. 

- Belmonte, Jose Christopher Y. 

- Belmonte, Ricardo Jr. T. 

- Benitez, Alfredo "Albee" B. 

- Bernos, Joseph Sto. Niño B. 

- Bertiz, Aniceto "John" III D. 

- Biazon, Rozzano Rufino B. 

- Billones, Emmanuel A. 

- Biron, Ferjenel G., M.D. 

- Bolilia, Lianda B. 

- Bondoc, Juan Pablo "Rimpy" P. 

- Bordado, Gabriel Jr. H. 

- Bravo, Anthony M., Ph.D. 

- Bravo, Maria Vida E. 

- Brosas, Arlene D. 

- Bulut-Begtang, Eleanor C. 

- Cagas, Mercedes C. 

- Calderon, Peter John D. 

- Calixto-Rubiano, Emi G. 

- Caminero, Wilfredo S. 

- Campos, Luis Jose Angel Jr. N. 

- Canama, Sabiniano S. 

- Cari, Jose Carlos L. 

- Casilao, Ariel 'Ka Ayik' B. 

- Castelo, Winston "Winnie" 

- Castro, France L. 

- Castro, Fredenil H. 

- Catamco, Nancy A. 

- Cayetano, Pia S. 

- Celeste, Jesus "Boying" F. 

- Cerafica, Arnel M. 

- Cerilles, Aurora Enerio 

- Chavez, Cecilia Leonila V. 

- Chipeco, Joaquin Jr. M. 

- Co, Christopher S. 

- Cojuangco, Carlos O. 

- Collantes, Ma. Theresa V. 

- Cortes, Jonas C. 

- Cortuna, Julieta R. 

- Cosalan, Ronald M. 

- Crisologo, Vincent "Bingbong" P. 

- Cua, Dakila Carlo E. 

- Cuaresma, Luisa Lloren 

- Cueva, Leo Rafael M. 

- Dalipe, Manuel Jose "Mannix" M. 

- Dalog, Maximo B. 

- Daza, Raul A. 

- De Jesus, Emmi A. 

- De Venecia, Christopher "Toff" Vera Perez 

- De Vera, Eugene Michael B. 

- Defensor, Arthur Jr. R. 

- Del Mar, Raul V. 

- Del Rosario, Manuel Monsour III T. 

- Deloso-Montalla, Cheryl P. 

- Dimaporo, Abdullah D. 

- Dimaporo, Mohamad Khalid Q. 

- Duavit, Michael John R. 

- Durano, Ramon "Red" VI H. 

- Dy, Napoleon S. 

- Elago, Sarah Jane I. 

- Enverga, Anna Katrina "Trina" M. 

- Erice, Edgar R. 

- Eriguel, Sandra Y., M.D. 

- Ermita-Buhain, Elenita Milagros "Eileen" 

- Escudero, Evelina G. 

- Espina, Rogelio J.,M.D. 

- Espino, Amado Jr. T. 

- Estrella, Conrado III M. 

- Eusebio, Richard C. 

- Evardone, Ben P. 

- Fariñas, Rodolfo C. 

- Fernando, Bayani F. 

- Ferrer, Juliet Marie D. 

- Ferrer, Luis "Jon-Jon" IV A. 

- Ferriol-Pascual, Abigail Faye C. 

- Floirendo, Antonio Jr. R. 

- Flores, Florencio Jr. T., M.D. 

- Fortun, Lawrence Lemuel H. 

- Fortuno, Salvio B. 

- Fuentebella, Arnulfo P. 

- Garbin, Alfredo Jr. A. 

- Garcia, Gwendolyn F. 

- Garcia, Jose Enrique III "Joet" S. 

- Garcia-Albano, Mylene J. 

- Garin, Oscar "Richard" Jr. S. 

- Garin, Sharon S. 

- Gasataya, Greg G. 

- Gatchalian, Wes 

- Geron, Rico B. 

- Go, Ana Cristina Siquian 

- Go, Mark O. 

- Gomez, Lucy T. 

- Gonzaga, Ruwel Peter S. 

- Gonzales, Alexandria P. 

- Gonzales, Aurelio Jr. "Dong" D. 

- Gonzalez, Fernando V. 

- Gorriceta, Arcadio H. 

- Gullas, Gerald Anthony Jr. V. 

- Hernandez, Ferdinand L. 

- Herrera-Dy, Bernadette "BH" 

- Hofer, Ann K. 

- Jalosjos, Seth Frederick P. 

- Javier, Paolo S. 

- Kho, Elisa "Olga" T. 

- Khonghun, Jeffrey D. 

- Labadlabad, Glona G. 

- Lacson, Virgilio S. 

- Lagman, Edcel C. 

- Lanete, Scott Davies S., M.D. 

- Laogan, Dennis C. 

- Lazatin, Carmelo II "Jon" B. 

- Leachon, Doy C. 

- Lee, Delphine "Delph" Gan 

- Limkaichong, Jocelyn Sy 

- Lobregat, Celso L. 

- Loong, Tupay T. 

- Lopez, Benhur Jr. B. 

- Lopez, Carlo V. 

- Lopez, Manuel Luis T. 

- Loyola, Roy M. 

- Macapagal-Arroyo, Gloria 

- Maceda, Edward Vera Perez 

- Madrona, Emmanuel F. 

- Malapitan, Dale "Along" R. 

- Manalo, Jesulito A. 

- Mangaoang, Allen Jesse C. 

- Mangudadatu, Suharto T. 

- Mangudadatu, Zajid G. 

- Marcoleta, Rodante D. 

- Marcos, Imelda R. 

- Mariño, Mario Vittorio "Marvey" A. 

- Marquez, Carlito S. 

- Martinez, Eric M. 

- Matugas, Francisco Jose II F. 

- Mellana, Evelyn P. 

- Mending, Makmod Jr. D. 

- Mercado, Roger G. 

- Mirasol, Alejandro Y. 

- Montoro, Teodoro "Ted" G. 

- Nava, Ma. Lucille L., M.D. 

- Nieto, John Marvin "Yul Servo" C. 

- Noel, Victoria Isabel G. 

- Nograles, Jericho Jonas B. 

- Nograles, Karlo Alexei B. 

- Nolasco, Ramon C. 

- Nuñez-Malanyaon, Corazon N. 

- Oaminal, Henry S. 

- Ocampo, Rosenda Ann 

- Olivarez, Eric L. 

- Ong, Edwin C. 

- Ong, Henry C. 

- Ortega, Pablo C. 

- Ortega, Vini Nola A. 

- Pacquiao, Rogelio "Ruel" D. 

- Paduano, Joseph Stephen S. 

- Palma, Wilter "Sharky" II Wee 

- Pancho, Gavini C. 

- Panganiban, Jose Jr. T. 

- Panotes, Marisol C. 

- Papandayan, Mauyag Jr. "Jun" B. 

- Pichay, Prospero Jr. A. 

- Pimentel, Johnny Ty 

- Pineda, Enrico A. 

- Plaza, Maria Valentina G. 

- Primicias-Agabas, Marlyn L. 

- Quimbo, Romero "Miro" S. 

- Radaza, Aileen C. 

- Ramos, Deogracias Jr. B. 

- Relampagos, Rene L. 

- Revilla, Strike B. 

- Roa-Puno, Cristina "Chiqui" 

- Robes, Florida "Rida" P. 

- Rocamora, Ramon V. A."Rav" 

- Rodriguez, Isidro Jr. S. 

- Rodriguez, Maximo Jr. B. 

- Roman, Geraldine B. 

- Romero, Michael L., PhD. 

- Romualdez, Yedda Marie K. 

- Romualdo, Xavier Jesus "XJ" D. 

- Roque, Harry Jr. L. 

- Roque, Rogelio Neil Pepito 

- Sacdalan, Jesus N. 

- Sagarbarria, Manuel T. 

- Sahali, Ruby M. 

- Salceda, Joey Sarte 

- Salimbangon, Benhur L. 

- Salo, Ron P. 

- Salon, Orestes T. 

- Sambar, Mark Aeron H. 

- Sandoval, Federico II "Ricky" S. 

- Santos-Recto, Vilma 

- Sarmiento, Cesar V. 

- Sarmiento, Edgar Mary S. 

- Sato, Josephine R. 

- Savellano, Deogracias Victor "DV" B. 

- Sema, Bai Sandra A. 

- Siao, Frederick W. 

- Silverio, Lorna C. 

- Singson, Eric D. 

- Suansing, Estrellita B. 

- Suansing, Horacio Jr. P. 

- Suarez, Danilo E. 

- Sy-Alvarado, Jose Antonio "Kuya Jonathan" R. 

- Tambunting, Gus S. 

- Tan, Angelina "Helen" DL, M.D. 

- Tan, Milagrosa T. 

- Tan, Shernee Abubakar 

- Tejada, Jose I. 

- Teves, Arnolfo Jr. "Arnie" A. 

- Tiangco, Tobias "Toby" M. 

- Ting, Randolph S. 

- Tinio, Antonio L. 

- Tolentino, Abraham "Bambol" N. 

- Treñas, Jerry P. 

- Tugna, Sherwin N. 

- Tupas, Raul "Boboy" C. 

- Turabin-Hataman, Sitti Djalia A. 

- Ty, Arnel U. 

- Umali, Reynaldo V. 

- Unabia, Peter "Sr. Pedro" M. 

- Ungab, Alberto T. 

- Unico, Renato Jr. J. 

- Uy, Juliette T. 

- Uy, Rolando "Klarex" A. 

- Uybarreta, Carlos Roman 

- Vargas, Alfredo III D. 

- Vargas-Alfonso, Baby Aline 

- Velarde, Mariano Michael Jr. M. 

- Velasco, Lord Allan Jay Q. 

- Velasco-Catera, Tricia Nicole Q. 

- Veloso, Vicente "Ching" S.E. 

- Vergara, Rosanna "Ria" V. 

- Villafuerte, Luis Raymund Jr. F. 

- Villanueva, Noel L. 

- Villar, Mark A. 

- Villaraza-Suarez, Anna Marie 

- Villarica, Linabelle Ruth R. 

- Villarin, Tom S. 

- Violago, Micaela S. 

- Yap, Arthur C. 

- Yap, Melecio Jr. J. 

- Yap, Victor A. 

- Yu, Divina Grace C. 

- Zamora, Maria Carmen S. 

- Zamora, Ronaldo B. 

- Zarate, Carlos Isagani T. 

- Zubiri, Manuel F. 




### Removed

No memberships removed


## Terms

### Added



- `term/17` - 17th Congress of the Philippines



### Removed


No terms removed


## Elections

### Added


No elections added


### Removed


No elections removed

octopusinvitro commented:
@tmtmtmtm I rebased this againg to reword the commit you mentioned and to fix some conflicts with master that appeared today. 

I also made a guess about which parts of the check-list you wanted in the PR description, let me know if that is what you wanted or I should add something else.

tmtmtmtm commented:
@octopusinvitro the first section of the checklist doesn't seem relevant to this PR. It would be good if you could also add an Issue for mapping these new Parties to Wikidata.  

@davewhiteland can you update the master checklist to include a step for Wikidata mapping of new Parties (or opening an Issue for it), as with mapping new People.
